src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -110,30 +136,60 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
     navigateTo(`/settings/${params.orgId}/projects/:projectId/alerts/new/`, router);
   };
 
-  renderEmpty() {
+  tryRenderFirstVisit() {
+    const {firstVisitShown} = this.state;
+
+    if (!firstVisitShown) {
+      return null;
+    }
+
+    return (
+      <WelcomeEmptyMessage
+        leftAligned
+        size="medium"
```

Comment:
I noticed that we use "sm, md, lg....." maybe you should rename it to `md` :)

Reply:
Will do in a separate PR :)